XLForm/XL/Descriptors/XLFormRowDescriptor.m

```diff
@@ -135,6 +135,12 @@ -(XLFormBaseCell *)cellForFormController:(XLFormViewController * __unused)formCo
             } else {
                 bundle = [NSBundle bundleForClass:NSClassFromString(cellClass)];
                 cellResource = cellClassString;
+                if ([cellClassString rangeOfString:@"."].location != NSNotFound) {
+                    NSArray *components = [cellClassString componentsSeparatedByString:@"."];
+                    cellResource = [components lastObject];
+                } else {
```

Comment:
The else case is not needed as it is the same as the default

Reply:
My bad, I forgot to remove the line **137**...